lib/vdsm/common/glob.py

```diff
@@ -1,60 +1,7 @@
-#
+# Copyright (c) 2001,
 # Copyright (c) 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009,
-# 2010, 2011, 2012, 2013, 2014, 2015 Python Software Foundation; All
-# Rights Reserved
```

Comment:
This copyright should not be removed.

Reply:
This should be a range... But anyway, yes, this will have to be changed manually to re-add those lines.

Reply:
Wait, this function was needed for python2, and we already have it in python3. This should be removed and the code should use the standard glob function instead, and remove the tests for backported function.

Reply:
Ok, let me see if I can post it fast, it does sound relatively easy.